apps/omg_child_chain/lib/omg_child_chain/release_tasks/set_block_submit_gas_price_strategy.ex

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019-2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.ChildChain.ReleaseTasks.SetBlockSubmitGasPriceStrategy do
```

Comment:
perhaps, if you don't want to deal with the release tasks, you could simply put this into release.exs. I think you've noticed I'm doing this for chch when it comes to simple configuration.